mosdef_cassandra/runners/runners.py

```diff
@@ -184,8 +178,11 @@ def _run_cassandra(cassandra, inp_file, log_file):
         )
         log.write(err)
 
-    if p.returncode != 0 or "error" in err.lower():
-        print("Cassandra error, see {}".format(log_file))
+    if p.returncode != 0 or "error" in err.lower() or "error" in out.lower():
+        raise CassandraRuntimeError(
```

Comment:
This makes it much more obvious when a simulation has failed, I like this a lot.